src/Testing/src/xunit/AspNetTestInvoker.cs

```diff
@@ -59,6 +68,43 @@ protected override async Task<decimal> InvokeTestMethodAsync(object testClassIns
             return time;
         }
 
+        protected async Task<decimal> RetryAsync(RetryAttribute retryAttribute, object testClassInstance)
+        {
+            var attempts = 0;
+            var timeTaken = 0.0M;
+            for (attempts = 0; attempts < retryAttribute.MaxRetries; attempts++)
+            {
+                timeTaken = await base.InvokeTestMethodAsync(testClassInstance);
+                if (!Aggregator.HasExceptions)
```

Comment:
Is there a way to get a copy of the first iterations error, and report that if the overall test fails?

There is always the chance that a test fails because it is run twice, and we want the details of the original error.

Reply:
Right now you actually get all 3 errors in the logs, due to how we output things, so in all practical purposes you get everything (at least in the template tests which have very very verbose logs)

here's an example where you can see all 3 failures, if you search for "Template directory already exists, deleting contents" you can see the retrys

Reply:
Ok! Sounds good.